Source/monster.cpp

```diff
@@ -1372,6 +1372,8 @@ bool MonsterWalk(int i, MonsterMode variant)
 			dFlags[monster.position.temp.x][monster.position.temp.y] &= ~BFLAG_MONSTLR;
 			dMonster[monster.position.tile.x][monster.position.tile.y] = i + 1;
 			break;
+		default:
+			break;

```

Comment:
What's the purpose of this?
```suggestion
```

Reply:
Exhaustive enum case handeling, like everywhere else so that we don't get a long ass compiler warning for every enum value not handled (monster mode does not just contain walk directions)